fastify.js

```diff
@@ -835,15 +860,17 @@ function build (options) {
       this[kLogLevel]
     )
 
-    app.once('preReady', () => {
+    avvio.once('preReady', () => {
       const context = this[kFourOhFourContext]
 
       const onRequest = this[kHooks].onRequest
-      const preHandler = this[kHooks].preHandler.concat(opts.beforeHandler || [])
+      const preValidation = this[kHooks].preValidation.concat(opts.preValidation || [])
```

Comment:
These `preValidation` hooks aren't being bound to the fastify instance, but the ones for normal routes are. Is this intended?

Reply:
It is not :D